engine/src/main/java/com/redhat/cloud/notifications/events/FromOpenBridgeHistoryFiller.java

```diff
@@ -103,23 +100,27 @@ public void execute() {
         }
 
         for (ProcessingError pe : items) {
+            try {
```

Comment:
Each error needs to be processed in its own try/catch block to make sure the loop is never broken by an exception.